scheduler-server/scheduling-dsl-compiler/src/libs/mission-model-generated-code.ts

```diff
@@ -9,3 +9,8 @@ export enum ActivityType {
   // This indicates to the compiler that we are using a string enum so we can assign it to string for our AST
   _ = '_',
 }
+
+export type Resource =
```

Comment:
I think this would be a prime candidate for an enum so that they can have a good autocompletion for resources `Resource.someResource`

Reply:
Resource is now a string enum 👍